backend/__tests__/activity.js

```diff
@@ -0,0 +1,19 @@
+import {Activity} from "../enums/activity";
+
+describe('Activity', () => {
+    describe('convertToActivity', () => {
```

Comment:
Wouldn't it be better to call the /users/:username/activity endpoint directly? Currently it looks like the test is not really testing anything relevant.

Reply:
Allg I added a test for the endpoint, but I think we should keep any extra tests for any helper functions we implement so that if our main endpoint test fails, we know if it's our helper function or something else

Reply:
Ok